onboard-jmuniz/app.json

```diff
@@ -30,7 +31,13 @@
     },
     "plugins": [
       "expo-router",
-      "expo-font"
+      "expo-font",
+      [
+        "expo-secure-store",
+        {
+          "faceIDPermission": "Allow $(PRODUCT_NAME) to access your Face ID biometric data."
```

Comment:
ah aqui a gente colocou quando tava seguindo a doc e tava com problema mas provavelmente nao precisa, mas de qualquer forma se for deixar teria que trocar o product_name pra onboard-jmuniz